app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -228,6 +230,11 @@ class AvatarView @JvmOverloads constructor(
         binding.avatarImage.baseLoadAvatar(correspondent = mergedContact ?: correspondent)
     }
 
+    private fun loadUserAvatar(correspondent: Correspondent, bimi: Bimi?) {
+        state.update(correspondent, bimi)
+        binding.avatarImage.baseLoadAvatar(correspondent)
+    }
```

Comment:
Why do we need two methods to load a user avatar? Here you defined `loadUserAvatar()` and somewhere higher in the class we have `loadAvatar(user: User)`